### ISSUE TITLE
JVM crash during bulk insert

### ISSUE THREAD

**Issue by piotr-pawlaczek** [questdb/questdb#1331]
**Describe the bug**
During bulk insert through InfluxDB wire I get JVM crash:
```
2021-09-15T20:25:00.915760Z I i.q.c.l.t.LineTcpMeasurementScheduler assigned trade1 to thread 0
2021-09-15T20:25:00.915856Z I i.q.c.l.t.LineTcpMeasurementScheduler network IO thread using table [workerId=0, tableName=trade1, nNetworkIoWorkers=1]
2021-09-15T20:25:00.929593Z I i.q.c.l.t.LineTcpMeasurementScheduler assigned table to writer thread [tableName=trade1, threadId=0]
2021-09-15T20:25:00.929736Z I i.q.c.p.WriterPool open [table=`trade1`, thread=15]
2021-09-15T20:25:00.929917Z I i.q.c.TableWriter open 'trade1'
2021-09-15T20:25:00.935944Z I i.q.c.TableWriter purging non attached partitions [path=/root/.questdb/db/trade1]
2021-09-15T20:25:00.937485Z I i.q.c.TableWriter removed partition dir: /root/.questdb/db/trade1/2021-06-01
2021-09-15T20:25:00.937572Z I i.q.c.p.WriterPool >> [table=`trade1`, thread=15]
2021-09-15T20:25:00.953568Z I i.q.c.TableWriter switched partition [path='/root/.questdb/db/trade1/2021-06-01']
2021-09-15T20:25:01.017186Z I i.q.c.l.t.LineTcpMeasurementScheduler network IO thread using table [workerId=1, tableName=trade1, nNetworkIoWorkers=2]
2021-09-15T20:25:01.021159Z I i.q.c.TableWriter switched to o3 [table=trade1]
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007f5c7fc034cf, pid=1, tid=23
#
# JRE version: OpenJDK Runtime Environment (11.0.7+10) (build 11.0.7+10)
# Java VM: OpenJDK 64-Bit Server VM (11.0.7+10, mixed mode, tiered, compressed oops, serial gc, linux-amd64)
# Problematic frame:
# J 1821 c2 sun.misc.Unsafe.getLong(J)J jdk.unsupported@11.0.7 (8 bytes) @ 0x00007f5c7fc034cf [0x00007f5c7fc034a0+0x000000000000002f]
#
# Core dump will be written. Default location: /root/.questdb/core.1
#
# An error report file with more information is saved as:
# /root/.questdb/hs_err_pid1.log
Compiled method (c2)  116990 1821       4       sun.misc.Unsafe::getLong (8 bytes)
 total in heap  [0x00007f5c7fc03310,0x00007f5c7fc03548] = 568
 relocation     [0x00007f5c7fc03488,0x00007f5c7fc03498] = 16
 main code      [0x00007f5c7fc034a0,0x00007f5c7fc03500] = 96
 stub code      [0x00007f5c7fc03500,0x00007f5c7fc03518] = 24
 scopes data    [0x00007f5c7fc03518,0x00007f5c7fc03520] = 8
 scopes pcs     [0x00007f5c7fc03520,0x00007f5c7fc03540] = 32
 dependencies   [0x00007f5c7fc03540,0x00007f5c7fc03548] = 8
Could not load hsdis-amd64.so; library not loadable; PrintAssembly is disabled
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
#

[error occurred during error reporting (), id 0xb, SIGSEGV (0xb) at pc=0x00007f5c8fc2c529]
```

**To Reproduce**
I try to do bulk insert. In the code below I flush every 50 records.

```java
dbPoints.add(dataPoint);
if (dbPoints.size() >= 50) {
    for (final InfluxDBPoint dbPoint : dbPoints) {
        sender.metric("trade1");
        final Map<String, String> tags = dbPoint.getTags();
        for (final Map.Entry<String, String> entry : tags.entrySet()) {
            sender.tag(entry.getKey(), entry.getValue());
        }
        final Map<String, Object> fields = dbPoint.getFields();
        for (final Map.Entry<String, Object> entry : fields.entrySet()) {
            if (entry.getValue() instanceof String) {
                sender.field(entry.getKey(), ((String) entry.getValue()).replace(" ", "_"));
            } else if (entry.getValue() instanceof Integer) {
                sender.field(entry.getKey(), (Integer) entry.getValue());
            } else if (entry.getValue() instanceof Long) {
                sender.field(entry.getKey(), (Long) entry.getValue());
            } else if (entry.getValue() instanceof Float) {
                sender.field(entry.getKey(), (Float) entry.getValue());
            } else if (entry.getValue() instanceof Double) {
                sender.field(entry.getKey(), (Double) entry.getValue());
            } else if (entry.getValue() instanceof Boolean) {
                sender.field(entry.getKey(), (Boolean) entry.getValue());
            } else if (entry.getValue() == null) {
                //skip
            } else {
                throw new IllegalArgumentException("Unknown value for key: " + entry.getKey() + "value: " + entry.getValue() + " class: " + entry.getValue().getClass());
            }
        }
        sender.$(dataPoint.getTimestamp());
    }
    sender.flush();
    dbPoints.clear();
}
```

**Expected behavior**
I expect data to be persisted. Table and column files are created on disk but `select * from trade1` returns nothing.

**Environment (please complete the following information):**
Helm chart version  6.0.3
additional config:

```
shared.worker.count: 2
pg.worker.count: 4
pg.worker.affinity: 1,2,3,4
cairo.sql.append.page.size: 256
```
I attach hs_err_pid1.log: [hs_err_pid1.log](https://github.com/questdb/questdb/files/7195590/hs_err_pid1.log)

**Comment by bsmth** [comment]
I've edited the original issue description to include the logs as an attachment to improve readability / discussion on this issue

**Comment by ideoma** [comment]
Fixed via #1342, to be released in 6.0.6
